### DISCUSSION TITLE
无需鉴权即可串流网盘内容

### DISCUSSION THREAD

**Discussion by Westwall98** [AlistGo/alist#2513] [Q&A]
如果在guest视角内设置起始页为/tmp，那么/root是需要登录管理员账号来访问的，我也用guest访问过了显示"object not found"，但是如果用一台没有登录过alist的设备使用vlc串流根目录的视频，无需鉴权即可串流，相当于访问到了文件

是我的操作方式有问题吗？还是设定上就是可以直接串流网盘内容的？

**Comment by ascdev-jia** [comment]
本人也有同样的疑问，WebDav绑定的网盘内容即使没有登录的情况下也是可以访问的，没有鉴权操作？
